backend/db/migrations/000163_add_agent_update_settings.down.sql

```
@@ -0,0 +1,6 @@
+DELETE FROM system_settings WHERE key IN (
```

SQLFluff (4.2.1)

[error] The 'WHERE' keyword should always start a new line.

(LT14)